data/publiccloud/restart_instance.sh

```diff
@@ -43,24 +43,28 @@ INSTANCE_ID=$2
 HOST=$3
 ZONE=$4
 CNT=120;
+LOG_SCRIPT="./log_instance.sh"
```

Comment:
by design we are assuming that restart_instanace.sh and log_instance.sh are in the same directory.. but shall we  also use full path here just in case? 

    /root/log_instance.sh

Reply:
What do you think of having some kind of `/data/` folder which is a copy of `data/pubiccloud` and put it into `PATH`. So we do not need to care about locations of helper scripts?

Reply:
nice !

Reply:
I like that. And ok to have it in another PR.